libs/query-engine-common/Cargo.toml

```diff
@@ -7,15 +7,15 @@ edition = "2021"
 
 [dependencies]
 thiserror = "1"
-url = "2"
+url.workspace = true
 query-connector = { path = "../../query-engine/connectors/query-connector" }
 query-core = { path = "../../query-engine/core" }
 user-facing-errors = { path = "../user-facing-errors" }
 serde_json.workspace = true
 serde.workspace = true
 connection-string.workspace = true
 psl.workspace = true
-async-trait = "0.1"
+async-trait.workspace = true
 tracing = "0.1"
```

Comment:
tracing related deps could probably be moved to workspace deps too?

Reply:
Done